### PR TITLE
Revert import statement updates of ConstraintLayout.java

### PR DESCRIPTION
**Pull request by jswong65**
This reverts import statement updates of *ConstraintLayout.java* in commit 9135d33c51be4816d2947d1f3d490e309aac4d3f.

For some reason the changes made in *ConstraintLayout.java* cause a build issue. Revert the changes of the import statement for now till we identify the root cause.

### CODE REVIEW

**Comment by oscar-ad** [comment]
> For some reason the changes made in ConstraintLayout.java cause a build issue.

What's the build issue?

**Comment by jswong65** [comment]
The **core / Assemble** check was failed starting with this change - https://github.com/androidx/constraintlayout/actions/runs/1862981219

image

By reverting the import statement changes, the **core / Assemble** check passed again (see the passed check in this PR)

File: constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java [hunk 1]
```diff
@@ -16,13 +16,6 @@
 
 package androidx.constraintlayout.widget;
 
-import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
-import static android.view.ViewGroup.LayoutParams.WRAP_CONTENT;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.MATCH_CONSTRAINT_SPREAD;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.MATCH_CONSTRAINT_WRAP;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.PARENT_ID;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.UNSET;
```
Comment (oscar-ad) [inline constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java:24]: Seems like a bug from importing a static class before the class it depends on.
ie:
Importing these bellow
```
import android.view.View;
import android.view.ViewGroup;
```
Should work

Comment (oscar-ad) [reply constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java:24]: Alternatively removing these static imports and calling the constants by the class should also do the trick.

Ie: Change mentions of
```Java
MATCH_CONSTRAINT_SPREAD
MATCH_CONSTRAINT_WRAP
PARENT_ID
UNSET
```
To
```Java
LayoutParams.MATCH_CONSTRAINT_SPREAD
LayoutParams.MATCH_CONSTRAINT_WRAP
LayoutParams.PARENT_ID
LayoutParams.UNSET
```

**Review by oscar-ad — APPROVED** [review]

File: constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java (hunk 1 above)
Comment (jswong65) [reply constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java:24]: Doing a quick pass for the easy fix now. Will revisit this issue later. Thanks for the suggestions!

**Pull request merged**